### PR TITLE
Update to newest version of argo-checkout packages

### PR DESCRIPTION
**Pull request by lemonmade**
This PR updates the Argo template to include the new packages. This implementation is a bit awkward since I install both the argo-checkout and argo-post-purchase packages in the template, but we do not have any infrastructure in the template to conditionally install dependencies (including existing conditional dependencies, like React), so I just left the broken window there for now.

### CODE REVIEW

File: scripts/generate/clean-up.ts [hunk 1]
```diff
@@ -2,7 +2,7 @@ import fs from 'fs';
 import path from 'path';
 import {exec} from 'child_process';
 
-export function cleanUp() {
+export function cleanUp(type: string) {
```
Comment (tleunen) [inline scripts/generate/clean-up.ts:5]: Not part of this PR obviously, but I wonder if the cleanup task should actually hard remove `node_modules` and reinstall them based on the "new" `package.json` to make sure only the relevant dependencies are installed.

**Review by tleunen — APPROVED** [review]

File: scripts/generate/clean-up.ts (hunk 1 above)
Comment (lemonmade) [reply scripts/generate/clean-up.ts:5]: Agreed, I think right now you'd get a diff if you `yarn install`ed after committing our initial template project, which is not nice :/
